test_data.yml

```diff
@@ -0,0 +1,29 @@
+barley_type: spring
```

Comment:
This file is usually not committed as it's deleted by the test's teardown function
```   
@classmethod
    def tearDownClass(cls):
        os.remove(cls.test_file)```